packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -11,19 +11,29 @@ interface ComboboxContentProps {
   readonly options: string[];
 
   /**
-   * onSelection handler.
+   * onSelect handler.
    */
-  readonly onSelection: (selection: string) => void;
+  readonly onSelect: (selection: string) => void;
 
   /**
    * actions
    */
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * pre selected option
+   * @default ""
+   * @optional
+   * @type string
```

Comment:
hmm... we haven't added the whole @optional and @type before. mostly because typescript handles those for us automatically. Should we add those?

Also, calling this "pre selected option" isn't really true in a sense that it is the selected option whenever the user interacts with it (assuming it is implemented correctly)

Maybe

> The selected value on the combobox content

or something